### DISCUSSION TITLE
Is there a recommended way to show and hide code ?

### DISCUSSION THREAD

**Discussion by F-one-1** [vuepress/core#536] [Q&A] [answered]
In the vuepress,I use  vue-highlight.js. 

![f1c341a0273a9a109d9dd94f552cbb3](https://user-images.githubusercontent.com/68687740/143671020-7141c27d-232e-4d0d-a56c-6a23990b23c1.png)

![ced996eab966c20812e371cebc57f56](https://user-images.githubusercontent.com/68687740/143671324-e94a6199-253e-412c-8951-2cf53f0eed56.png)

**but it may not ok in the vuepress-next. Can you recommend some tool to use in it.Or what similar tools are you using to implement a similar approach**

**Comment by meteorlxy** [comment, marked as answer]
If vue-highlight.js supports vue 3, you can use it in vuepress-next for sure.

**Comment by F-one-1** [comment]
Year,thanks.I have solved it
